meta-sokol-flex-staging/dynamic-layers/openembedded-layer/recipes-support/pidgin/pidgin_%.bbappend

```diff
@@ -0,0 +1,5 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: GPL-2.0
+# ---------------------------------------------------------------------------------------------------------------------
+
+RDEPENDS:${PN}:append = " python3"
```

Comment:
@muhamzai I have worked with @akif-tariq to debug and determine that we need to set the runtime dependency for the finch package, not the ${PN}.
Can you please update this as `RDEPENDS:finch:append = " python3"`

Reply:
@kergoth or should it be `RDEPENDS:finch:append = " python3-core"` ?

Reply:
@muhamzai I think it should be `RDEPENDS:finch:append = " python3-core"` like we saw in the bluez recipe.

Reply:
@ArsalanHAwan updated the PR as the convention we saw in bluez5 
@kergoth kindly review